pages/common/john.md

```diff
@@ -25,12 +25,12 @@
 
 - Crack password hashes, using a specific hash format:
 
-`john --format={{md5crypt|...}} {{path/to/hashes.txt}}`
+`john --format={{hash_format}} {{path/to/hashes.txt}}`
```

Comment:
```suggestion
`john --format={{md5crypt}} {{path/to/hashes.txt}}`
```

Reply:
@zdroid I think @agnivade  is right ; `hash_format` is more generic and anyway, another example is provided for listing possible hash formats. So, specifying a particular hash format seems confusing, I think.

Reply:
@dhondta so you're saying that we should leave it as `hash_format`? I'd agree with that, but just want to make sure :P

Reply:
@sbrl Yes, that's indeed what I'm saying. Let us leave it as `hash_format`.

Reply:
But that's not what I said :) So your statement "agnivade is right" is not right :laughing: 

The reason I suggested to use `md5crypt` is because we prefer concrete examples rather than generic ones to clarify the command usage, and in fact, it is part of our style guide.

I am wondering why you think specifying a hash format can be confusing ? Yes, there is an example to list hash formats, and a concrete example can stay too.

The discussion whether md5crypt is an appropriate example or not can of course be debated, and if needed something else can be used. But following our usual guidelines, I am in favor of using a concrete example.

Reply:
@agnivade This looks like a case when two rights make one wrong. :laughing:

I'm for md5crypt being put here, as there is `--format` preceding it.

Reply:
Oops, I totally forgot that I'm the commit author here lol

PR updated